collections/_article/dev-snapshot-godot-4-4-beta-1.md

```diff
@@ -0,0 +1,289 @@
+- XR: Disable hand tracking by default. This has a non-trivial performance cost and is not needed for many games. Users need to enable the setting if they want to use hand tracking now. ([GH-95153](https://github.com/godotengine/godot/pull/95153))
+- Automatically resolve initial and final action for draw lists. `RenderingDevice.draw_list_begin()` has been vastly simplified and so it will be much easier to use. The old method signature will continue to work, but we recommend using the new method since it is much simpler. ([GH-98670](https://github.com/godotengine/godot/pull/98670))
+- Editor camera override has moved from the editor viewport to the "Game" tab. ([GH-97257](https://github.com/godotengine/godot/pull/97257))
+
+UIDs in particular are a pretty massive shake-up to existing project structures, and have understandably been among the more contentious changes of the 4.4 development cycle. We have a blogpost in the works to better explain the rationale behind that change, and the benefits such a system brings to the table, so please stay tuned for more info!
+
+### Animation
+One of our newest additions to the animator's toolkit is markers ([GH-91765](https://github.com/godotengine/godot/pull/91765)) by ChocolaMint. Markers allow you to create sub regions of an animation that can be jumped to, or looped without playing the entire animation. This functionality is even supported inside AnimationTree, where you can easily select animation markers for looping or to begin playback.
+
+We've also added `LookAtModifier3D` ([GH-98446](https://github.com/godotengine/godot/pull/98446)) to handle 3D model procedural animation, partially replacing the deprecated `SkeletonIK3D`. Thanks to the efforts of animation expert Silc Renew (Tokage), users no longer need to rely on specific bone structures and arbitrary layouts; this new tool allows for angle limitations, forward axis settings, etc., and is specialized for making a 3D character model look in the target direction.
+
+Sneaking in right before the feature freeze, the Animation team added `SpringBoneSimulator3D`([GH-101409](https://github.com/godotengine/godot/pull/101409)) as well. The spring bone is an open source multi-platform module that is distributed by the [VRM consortium](https://vrm-consortium.org/en/) under the MIT license. It has been available in Godot as an [add-on](https://godotengine.org/asset-library/asset/2031) for a while now, but there were some difficulties in stability and complex setup due to the complicated data structure. Tokage refined it based on `SkeletonModifier3D` with some improvements by leaning on existing core functionality, so it is now much improved and easier to use!
```

Comment:
```suggestion
Sneaking in right before the feature freeze, the Animation team added `SpringBoneSimulator3D`([GH-101409](https://github.com/godotengine/godot/pull/101409)) as well. The spring bone is an open source cross-platform module that is distributed by the [VRM consortium](https://vrm-consortium.org/en/) under the MIT license. It has been available in Godot as an [add-on](https://godotengine.org/asset-library/asset/2031) for a while now, but there were some difficulties in stability and complex setup due to the complicated data structure. Tokage refined it based on `SkeletonModifier3D` with some improvements by leaning on existing core functionality, so it is now much improved and easier to use!
```

Reply:
For consistency, I assume that `cross-platform` is used more often than `multi-platform` on the Godot website.